index.html

```diff
@@ -282,6 +282,57 @@ <h5>Skills</h5>
           </ul>
         </div>
       </section>
+
+      <section id="contact-section">
+        <div class="contact">
+          <h6>Contact me</h6>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. <br />
+            I’d love to help with it!
+          </p>
+          <form
```

Comment:
[OPTIONAL]
- [ ] please kindly use descriptive words as placeholders in your contact forms so that users of your portfolio can be exactly sure of what to imput in a agiving fiels. see the design template below as example reference :+1: 
![exam](https://user-images.githubusercontent.com/89055502/235815016-e853e953-370e-4816-9013-08d059ea1c37.png)